### PR TITLE
Implement ACLCustomMRU

### PR DESCRIPTION
**Pull request by learn-more**
## Purpose

JIRA issue: [CORE-9281](https://jira.reactos.org/browse/CORE-9281)

### CODE REVIEW

File: dll/win32/browseui/ACLCustomMRU.cpp [hunk 1]
```diff
@@ -0,0 +1,133 @@
...
+    if (Status != ERROR_SUCCESS)
+        return HRESULT_FROM_WIN32(Status);
+
+    m_MRUData.RemoveAll();
+    dwMax = max(0, dwMax);
+    dwMax = min(29, dwMax);
+    while (dwMax--)
+        m_MRUData.Add(CStringW());
+
+    WCHAR MRUList[40];
```
Comment (HBelusca) [inline dll/win32/browseui/ACLCustomMRU.cpp]: Is there any reason for the value 40, and not more or not less?

Comment (learn-more) [reply dll/win32/browseui/ACLCustomMRU.cpp]: Because 29 items is the max, and I took a bit extra space.

Comment (HBelusca) [reply dll/win32/browseui/ACLCustomMRU.cpp]: Ah ok; adding 1 short sentence comment about the 29 items max would be nie perhaps?

Comment (learn-more) [reply dll/win32/browseui/ACLCustomMRU.cpp]: 4 lines above this is the `max()` call, I think that's pretty self explanatory.

**Pull request merged**